cardano-cli/src/Cardano/CLI/Run/Ping.hs

```diff
@@ -4,32 +4,31 @@
 {- HLINT ignore "Move brackets to avoid $" -}
 
 module Cardano.CLI.Run.Ping
-  ( PingCmd(..)
-  , PingClientCmdError(..)
-  , renderPingClientCmdError
-  , runPingCmd
-  , parsePingCmd
-  ) where
-
-import           Cardano.Api.Pretty
+  ( PingCmd (..),
+    PingClientCmdError (..),
+    renderPingClientCmdError,
+    runPingCmd,
+    parsePingCmd,
+  )
+where
 
+import Cardano.Api.Pretty
 import qualified Cardano.Network.Ping as CNP
-
-import           Control.Applicative ((<|>))
-import           Control.Concurrent.Class.MonadSTM.Strict (StrictTMVar)
+import Control.Applicative ((<|>))
```

Comment:
Incorrect formatting. Can you run this through stylish: https://github.com/input-output-hk/cardano-node-wiki/wiki/Style-guide#code-linting-and-formatting-tools